doc/index.rst

```diff
@@ -1,34 +1,29 @@
 RISE
 ====
 
-**Reveal.js - Jupyter/IPython Slideshow Extension**, also known as *live_reveal*
-
-.. image:: https://badges.gitter.im/Join%20Chat.svg
-    :target: https://gitter.im/damianavila/RISE
-    :alt: Gitter
+**Reveal.js - Jupyter/IPython Slideshow Extension**
 
 What is RISE?
 -------------
-As you know... we love **Jupyter/IPython** and we like **Reveal.js** too.
-
-Previously, we developed a "converter" for the `nbconvert` library to
-export a ``ipynb`` file to a *STATIC* html slideshow based in **Reveal.js** library.
+As you know... we love **Jupyter** and we like **reveal.js** too.
 
-But with RISE, you don't have a *STATIC* version anymore, you have a **LIVE**
-version! A notebook **rendered** as a **Reveal.js**-based slideshow,
-where you can **execute** code or show to the audience whatever you can show/do
-inside the notebook itself (but in a "slidy" way).
+With RISE, a Jupyter notebook extension, you can instantly turn your
```

Comment:
Same comment as above, I would try to keep the word "live".

Reply:
done